### DISCUSSION TITLE
Audio Gridder crashes and close unexpectedly

### DISCUSSION THREAD

**Discussion by Linxz84** [apohl79/audiogridder#456] [Help]
I am starting to use audio gridder and I find that it constantly closes and reopens the audiogridder server

is there any way to fix this issue??

[crash report.zip](https://github.com/apohl79/audiogridder/files/6230060/crash.report.zip)

**Comment by CaBuffalo** [comment]
Hi, what's your server / client set up and which plugin(s) does it happen
when you were using?

On Tue, Mar 30, 2021 at 8:17 AM Linxz84 ***@***.***> wrote:

> I am starting to use audio gridder and I find that it constantly closes
> and reopens the audiogridder server
>
> is there any way to fix this issue??
>
> crash report.zip
> <https://github.com/apohl79/audiogridder/files/6230060/crash.report.zip>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/456>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DDZPMBUPZ63XQAEI3TGHTPNANCNFSM42CI7MMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Linxz84** [reply]
it happens with any plugin

![WhatsApp Image 2021-03-30 at 13 16 13](https://user-images.githubusercontent.com/74384853/113021760-483d6780-915a-11eb-8ee1-2e1f1582ecde.jpeg)
![WhatsApp Image 2021-03-30 at 13 16 13 (1)](https://user-images.githubusercontent.com/74384853/113021764-48d5fe00-915a-11eb-8dbd-295c5f772b10.jpeg)

![WhatsApp Image 2021-03-30 at 13 01 23](https://user-images.githubusercontent.com/74384853/113020045-981b2f00-9158-11eb-9a10-7239f31e6782.jpeg)

**Comment by CaBuffalo** [comment]
Sorry, I mean, what's your client machine, server machine and which daw are
you using? Is this MBP your DAW machine?

On Tue, Mar 30, 2021 at 9:17 AM Linxz84 ***@***.***> wrote:

> it happens with any plugin
>
> [image: WhatsApp Image 2021-03-30 at 13 16 13]
> <https://user-images.githubusercontent.com/74384853/113021760-483d6780-915a-11eb-8ee1-2e1f1582ecde.jpeg>
> [image: WhatsApp Image 2021-03-30 at 13 16 13 (1)]
> <https://user-images.githubusercontent.com/74384853/113021764-48d5fe00-915a-11eb-8dbd-295c5f772b10.jpeg>
>
> [image: WhatsApp Image 2021-03-30 at 13 01 23]
> <https://user-images.githubusercontent.com/74384853/113020045-981b2f00-9158-11eb-9a10-7239f31e6782.jpeg>
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/456#discussioncomment-548963>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6F7C632YWMR7Z2KSG3TGH2TFANCNFSM42CI7MMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Linxz84** [reply]
sorry I had not understood

Im using Pro tools 2018 on my Macbook pro 2018 Mojave OS
and the server is a Macbook pro 2013 i5 8gb Mavericks OS

**Comment by CaBuffalo** [comment]
I don't have PT. Can you try a different DAW to do a simple test?

Your systems are fine with AG. I doubt the problem is from either Mavericks
or PT

On Tue, Mar 30, 2021 at 9:48 AM Linxz ***@***.***> wrote:

> sorry I had not understood
>
> Im using Pro tools 2018 on my Macbook pro 2018 Mojave OS
> and the server is a Macbook pro 2013 i5 8gb Mavericks OS
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/456#discussioncomment-549289>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6DLWUV7UCHAX2WEX4DTGH6FLANCNFSM42CI7MMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Linxz84** [reply]
have you seen the crash report?

**Reply by Linxz84** [reply]
wich daw you recomend to test this?

**Comment by CaBuffalo** [comment]
I would say Reaper because it's available to trial.

On Tue, Mar 30, 2021 at 10:02 AM Linxz ***@***.***> wrote:

> wich daw you recomend to test this?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/456#discussioncomment-549353>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6H6MGTU4CYL4CEBEA3TGH733ANCNFSM42CI7MMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Linxz84** [reply]
II already tried reaper and it happens exactly the same

**Comment by apohl79** [comment]
As the server crashes the DAW does not not matter. Please follow https://audiogridder.com/bug-reports/ to provide more info. The crash report doesn't show enough insight. Thx.

**Reply by Linxz84** [reply]
Crash reports
[AudioGridderServer_.zip](https://github.com/apohl79/audiogridder/files/6231063/AudioGridderServer_.zip)

**Comment by Linxz84** [comment]
I ve found the solution, i changed the Screen capturing mode to legacy and that fix the issue.

**Reply by Linxz84** [reply]
the problem persists but not as often as before

**Reply by looshamik** [reply]
Hola. Veo en las capturas que enviaste de tu equipo que están en Español así que me dirigiré a ti, si no tienes inconveniente, en este idioma. Ha pasado ya un tiempo desde que abriste el hilo. Quería saber si llegaste a resolver el problema de manera estable. Recientemente estoy tratando de configurar un equipo similar utilizando AG-MPH (basado en AudioGridder para MP MIDI controller) y me estoy encontrando con el mismo problema insidioso que describes aquí.
Gracias

**Comment by CaBuffalo** [comment]
Awesome. Please keep trying. Enjoy.

On Tue, Mar 30, 2021 at 10:56 AM Linxz ***@***.***> wrote:

> I ve found the solution, i changed the Screen capturing mode to legacy and
> that fix the issue.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/456#discussioncomment-549559>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6HGWAEIYIBVD6SQ6IDTGIGFLANCNFSM42CI7MMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by CaBuffalo** [comment]
Can you find some log file the follow the bug return here?
https://audiogridder.com/bug-reports/

Thanks.

On Wed, Mar 31, 2021 at 3:02 AM Linxz ***@***.***> wrote:

> the problem persists but not as often as before
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/456#discussioncomment-552007>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6A75VDFWC57H6L2ANTTGLXLBANCNFSM42CI7MMQ>
> .
>


-- 
Kevin Ma
Supported by Gmail.
